packages/shared/src/components/auth/AuthOptionsInner.tsx

```diff
@@ -130,8 +130,9 @@ function AuthOptionsInner({
   );
   const { refetchBoot, user, isFunnel } = useAuthContext();
   const router = useRouter();
-  const isOnboardingPage = !!router?.pathname?.startsWith('/onboarding');
-  const isReorderExperiment = useFeature(featureOnboardingReorder);
+  const isOnboardingOrFunnel =
+    !!router?.pathname?.startsWith('/onboarding') || isFunnel;
```

Comment:
Maybe put this condition within the context itself so everything that uses this will get the same values.